.github/workflows/artifact-publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Java Package
+
+on:
+  push:
+    tags:
+      - 'jpo-s3-deposit-*'

```

Comment:
Did we want to ensure we only are creating artifact for "release" tags?

Reply:
I figured that we should go with the current convention in the USDOT repository. All of their release tags are prefixed with the name of the repository. Do we plan to create separate release tags in the USDOT branches?